config/settings_schema.json

```diff
@@ -167,7 +167,7 @@
       {
         "type": "range",
         "id": "spacing_grid_horizontal",
-        "min": 4,
+        "min": 0,
```

Comment:
Something I'm noticing for sliders is that we night need to adjust the width calculation of a slide to always show a peek effect: [video](https://screenshot.click/15-02-m811d-7huka.mp4)

Reply:
Looks we're adding a hardcoded value on mobile to always keep the grid peek: [screenshot](https://screenshot.click/15-11-8sulr-ktqdd.png)